tailwind.config.js

```diff
@@ -0,0 +1,27 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./index.html"],
+  theme: {
+    extend: {
+      fontFamily: {
+        arial: "arial",
+      },
+      colors: {
+        "button-gray": "#f8f9fa",
```

Comment:
There is a way you can delary all grays at the same time instead of repeating yourself, this may help you next time if you reuse you color
![code](https://user-images.githubusercontent.com/56281095/196453578-2403c818-1c1b-4f45-9a83-b10a0f5c337e.png)